src/operations/indexes.ts

```diff
@@ -95,6 +95,8 @@ export interface IndexDescription
   collation?: CollationOptions;
   name?: string;
   key: Document;
+  /* Specifies that this index is clustered. This is not a valid option to provide to 'createIndexes', but can appear in the options returned for an index via 'listIndexes'. */
+  clustered?: boolean;
```

Comment:
@baileympearson I'm trying to be careful here and not include the new boolean in CreateIndexesOptions (because you cannot create a clustered index normally) but only in IndexDescription.